orm/models.py

```diff
@@ -20,6 +22,8 @@
     "lte": "__le__",
 }
 
+MODEL = typing.TypeVar("MODEL", bound="Model")
```

Comment:
What value does this bring? I mean we could call bulk_update with `Model` itself. right?

Reply:
What do you mean? We use it as a type annotation for obj

Reply:
Yeah I understand. I mean you've done this:

```python
def bulk_update(self, objects: typing.List[MODEL], ...):
    ....
```

What would be the difference if we did:

```python
def bulk_update(self, objects: typing.List[Model], ...):
    ....
```

Reply:
In this case Model will be undefined because it has been defined after bulk_update

Reply:
Maybe "Model" ?